dist/api_data.py

```diff
@@ -1034,6 +1034,10 @@ def __ge__(self, other):
             @ref tune_durability for more information''',
             choices=['dsync', 'fsync', 'none']),
         ]),
+    Config('verify_metadata', 'false', r'''
+        open connection and verify any WiredTiger-owned table. This API
+        allows verification and detection of corruption in WiredTiger tables.''',
```

Comment:
I agree with the sentiment and made this change. But I'd like to find some phrase other than `WiredTiger metadata` because that is something very specific in my head.